src/contact.ts

```diff
@@ -110,6 +112,11 @@ export class Contact implements Sayable {
       star:       !!rawObj.StarFriend,
       stranger:   !!rawObj.stranger, // assign by injectio.js
       avatar:     rawObj.HeadImgUrl,
+      /**
+       * @see 1. https://github.com/nodeWechat/wechat4u/blob/master/src/interface/contact.js#L65
+       * @see 2. https://github.com/Urinx/WeixinBot/blob/master/README.md
+       */
+      brand:      !!rawObj.UserName && !rawObj.UserName.startsWith('@@') && (rawObj.VerifyFlag & 8) !== 0,
```

Comment:
```
// tslint:disable-next-line
```

will disables all rules for the following lines, it seems it's not a good choose for this case.

how about this?

```ts
!!rawObj.UserName && !rawObj.UserName.startsWith('@@') && [8, 24, 56].indexOf(rawObj.VerifyFlag) > -1,
```